code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -265,6 +265,11 @@
 	else
 		..()
 
+/mob/verb/osscheck()
+	set name = "osscheck"
+	set hidden = TRUE
+	alert(usr, "You are running code from an OSS codebase")
+

```

Comment:
this PR gives it away though.
you can also use the verb and search DM for the string it outputs.

tl;dr bad meme.